app/components/teachers_index_component.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <p><%= govuk_link_to("Find and claim a new ECT", new_ab_claim_an_ect_find_path) %></p>
+    <% if bulk_upload_enabled? %>
+      <p><%= govuk_link_to("Upload a CSV to record outcomes for multiple ECTs", new_ab_batch_action_path) %></p>
+      <p><%= govuk_link_to("Upload a CSV to claim multiple new ECTs", new_ab_batch_claim_path) %></p>
+    <% end %>
+    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+  </div>
+</div>
+
+<%= render TeachersIndex::HeaderSectionComponent.new(
+  status: status,
+  current_count: current_count,
+  open_count: open_count,
+  closed_count: closed_count,
+  query: query
+) %>
+
+<%= render TeachersIndex::SearchSectionComponent.new(status: status, query: query) %>
+
+<%= render TeachersIndex::TableSectionComponent.new(
+  teachers: teachers,
+  pagy: pagy,
+  status: status,
+  query: query
+) %>
```

Comment:
This bit feels a bit odd, the 'parent' component should have [child slots](https://viewcomponent.org/guide/slots.html).

So something like:

```
class Teachers::IndexComponent
  has_one :header
  has_one :search_box
  has_one :table
end
```